transaction/src/manifest/lexer.rs

```diff
@@ -3,9 +3,9 @@ use sbor::rust::str::FromStr;
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Span {
     /// The start of the span, inclusive
-    pub start: (usize, usize),
+    pub start: usize,
     /// The end of the span, inclusive
```

Comment:
We used to do self.current - 1, but we don't any more. So this should probably be `exclusive`.